src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDays = 7;
+  const shortTermDays = 3;
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
```

Comment:
Here you can simplify the naming a little.

Reply:
@roman-mirzoian 
  const longTerm = 7;
  const shortTerm = 3;
  const costPerDay = 40;
  const mediumDiscount = 20;
  const longDiscount = 50;
  const basePrice = costPerDay * days;
  
that is better?)